static/src/javascripts/projects/common/modules/crosswords/anagram-helper/main.js

```diff
@@ -0,0 +1,167 @@
+        if (this.canShuffle()) {
+            this.setState({
+                showInput: false,
+            });
+        }
+    },
+
+    reset() {
+        if (this.state.clueInput) {
+            this.setState({
+                clueInput: '',
+                showInput: true,
+            });
+        }
+    },
+
+    canShuffle() {
+        return this.state.clueInput && this.state.clueInput.length > 0;
+    },
+
+    render() {
+        const closeIcon = {
+            __html: inlineSvg('closeCentralIcon'),
+        };
+        const clue = helpers.getAnagramClueData(
+            this.props.entries,
+            this.props.focussedEntry
+        );
+        const cells = helpers.cellsForClue(
+            this.props.entries,
+            this.props.focussedEntry
+        );
+        const entries = cells.map(
+            coords => this.props.grid[coords.x][coords.y]
+        );
+
+        const letters = this.shuffleWord(this.state.clueInput, entries);
+
+        const inner = this.state.showInput
+            ? React.createElement(ClueInput, {
+                  value: this.state.clueInput,
+                  clue,
+                  onChange: this.onClueInput,
+                  onEnter: this.shuffle,
+              })
+            : React.createElement(Ring, {
+                  letters,
+              });
+
+        return React.createElement(
+            'div',
+            {
+                className: 'crossword__anagram-helper-outer',
+                'data-link-name': 'Anagram Helper',
+            },
+            React.createElement(
+                'div',
+                {
+                    className: 'crossword__anagram-helper-inner',
+                },
+                inner
+            ),
+            React.createElement('button', {
+                className: 'button button--large button--tertiary crossword__anagram-helper-close',
+                onClick: this.props.close,
+                dangerouslySetInnerHTML: closeIcon,
+                'data-link-name': 'Close',
+            }),
+            React.createElement(
+                'button',
+                {
+                    className: `button button--large ${!this.state.clueInput ? 'button--tertiary' : ''}`,
+                    onClick: this.reset,
+                    'data-link-name': 'Start Again',
+                },
+                'start again'
+            ),
+            React.createElement(
+                'button',
+                {
+                    className: `button button--large ${!this.canShuffle() ? 'button--tertiary' : ''}`,
+                    onClick: this.shuffle,
+                    'data-link-name': 'Shuffle',
+                },
+                'shuffle'
+            ),
+            React.createElement(CluePreview, {
+                clue,
+                entries,
+                letters,
+                hasShuffled: !this.state.showInput,
+            })
+        );
+    },
+});
+
+export default AnagramHelper;
```

Comment:
Is there a way to make the named export work from the legacy modules? If I use export { AnagramHelper } it looks like it can't be imported from the legacy classes.